### ISSUE TITLE
Items with the "/twin_blade/" tag aren't counted as Dual Wielding items

### ISSUE THREAD

**Issue by xeroplz** [aura-project/aura#362]
Items with the /twin_blade/ tag are supposed to count as being dual wielded weapons, even though they only go in the right hand. I had added the twin_blade check to isDualWielding in a pull request, but removed it as it complicated calculations down the line. I think it'd be best to leave this here as an issue.

**Comment by exectails** [comment]
What do you mean by complicating calculations?

**Comment by UnknownVoid** [comment]
I can't really see anything complicated...

Lets see from what i remember, duel Wielding is mostly a damage buff, other then simple equipment checks and some skill checks.

Perhaps there talking about the support for https://github.com/aura-project/aura/issues/7
which would be needed for duel wielding to work properly (I could be wrong about that but meh)

**Comment by xeroplz** [comment]
Here are a couple examples"
1. Attacking a dungeon orb, exception in GetLeftHandStaminaUsage (or something along those lines) because it expects a weapon to be in the left hand.
2. Second attack of combat mastery makes the enemy's HP bar go UP because I guess it expects something from the left hand again?
I mean these are pretty simple, but if they were any more small things like those that would need to be changed...

**Comment by exectails** [comment]
And what exactly do you mean by it's considered a dual wielding weapon? Does it hit twice? And is that all?

**Comment by xeroplz** [comment]
Yes, its combat performance is the same as normal dual wielding.

**Comment by exectails** [comment]
That's different from saying "it's considered dual wield" though. What we want is not to handle it just like dual wielding, we just want it to hit twice. I think we could make the loop run twice for that specific tag, without changing anything else, it's just a special case.

**Comment by xeroplz** [comment]
Would making it run twice fix things like attack animations for the weapon as well?

**Comment by exectails** [comment]
That depends on which options are set for that weapon. We just have to set the correct ones.
